### DISCUSSION TITLE
GLIBC_2.34 not found

### DISCUSSION THREAD

**Discussion by ghost** [AntiMicroX/antimicrox#689] [Q&A]
i get the error antimicrox: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by antimicrox) when ever i try to run antimicrox. I've seen many people encounting this error, but none seem to find a solution that works for me. Any thoughts?

**Comment by pktiuk** [comment]
I can't help you without any knowledge about your OS configuration and AntiMicroX version and package
